eav/registry.py

```diff
@@ -131,19 +131,22 @@ def _detach_manager(self):
 
     def _attach_signals(self):
         '''
-        Attach all signals for eav
+        Attach pre- and post- save signals from model class
+        to Entity helper. This way, Entity instance will be
+        able to prepare and clean-up before and after creation /
+        update of the user's model class instance.
         '''
-        post_init.connect(Registry.attach_eav_attr, sender=self.model_cls)
-        pre_save.connect(Entity.pre_save_handler, sender=self.model_cls)
-        post_save.connect(Entity.post_save_handler, sender=self.model_cls)
+        post_init.connect(Registry.attach_eav_attr, sender = self.model_cls)
```

Comment:
@IwoHerka you are not consistent in usage of this style. That's another case I believe that no whitespace surrounding assignments in function calls is inherently Pythonic, but whatever way you choose, I dislike inconsistency here.